risc0/zkvm/src/host/client/prove/mod.rs

```diff
@@ -196,19 +202,47 @@ impl Default for ProverOpts {
             prove_guest_errors: false,
             receipt_kind: ReceiptKind::Composite,
             control_ids: ALLOWED_CONTROL_IDS.to_vec(),
+            segment_limit_po2: 21,
```

Comment:
I suggest adding `DEFAULT_SEGMENT_LIMIT_PO2` constant

Reply:
That one is already taken, how about `DEFAULT_MAX_PO2`

Reply:
Addressed in 582001f5